src/Common/Enums.ts

```diff
@@ -62,8 +62,11 @@ export enum OCSDKTelemetryEvent {
   GETLWISTATUSSUCCEEDED = "GetLwiStatusSucceeded",
   GETLWISTATUSFAILED = "GetLwiStatusFailed",
   SESSIONINITSTARTED = "SessionInitStarted",
+  SESSIONINITV2STARTED = "SessionInitV2Started",
```

Comment:
Do we really needed a new marker? AFAIK we can differentiate between those calls with `requestPath`